src/doc/rustdoc/src/how-to-write-documentation.md

```diff
@@ -70,13 +156,18 @@ and finally provides a code example.
 
 `rustdoc` is using the [commonmark markdown specification]. You might be
 interested into taking a look at their website to see what's possible to do.
+ - [commonmark quick reference]
+ - [current spec]
 
-## Lints
-
-To be sure that you didn't miss any item without documentation or code examples,
-you can take a look at the rustdoc lints [here][rustdoc-lints].
```

Comment:
Hmm, is there a reason you removed this?

Reply:
I gave that section an entire chapter.   I moved it into the what-to-include section, although I rewrote it to have details.

Reply:
Oh nice! Much better :D